publish.sh

```diff
@@ -70,8 +70,10 @@ function pre_process {
     output="\ \ \ \ git:\n      url: https://github.com/flame-engine/flame.git\n      ref: main\n      path: packages/flame"
     find . -name "*.md" -exec sed -i "/<VERSION>.*/a $output" {} \;
     find . -name "*.md" -exec sed -i "s/ <VERSION>//" {} \;
+    find . -name "resources.md" -exec sed -i "s/--VERSION--/latest/" {} \;
   else
     find . -name "*.md" -exec sed -i "s/<VERSION>/$version/" {} \;
```

Comment:
Just change this to be the new placeholder instead, so that we don't have two placeholders for version

Reply:
Ok, let me find something that will be valid for both markdown and toctree restrictions.

Reply:
Done.